app/frontend/src/features/user/UserTextAndLabel.tsx

```diff
@@ -0,0 +1,63 @@
+import LabelAndText from "components/LabelAndText";
+import {
+  AGE_GENDER,
+  EDUCATION,
+  HOMETOWN,
+  JOINED,
+  LANGUAGES_FLUENT,
+  LAST_ACTIVE,
+  OCCUPATION,
+  REFERENCES,
+} from "features/constants";
+import {
+  LANGUAGES_FLUENT_FALSE,
+  LAST_ACTIVE_FALSE,
+} from "features/profile/constants";
+import { User } from "pb/api_pb";
+import { dateTimeFormatter, timestamp2Date } from "utils/date";
+import { timeAgo } from "utils/timeAgo";
+
+interface Props {
+  user: User.AsObject;
+}
+
+export const LabelsReferencesLastActive = ({ user }: Props) => (
+  <>
+    <LabelAndText label={REFERENCES} text={`${user.numReferences || 0}`} />
+    <LabelAndText
+      label={LAST_ACTIVE}
+      text={
+        user.lastActive
+          ? timeAgo(timestamp2Date(user.lastActive))
+          : LAST_ACTIVE_FALSE
+      }
+    />
+  </>
+);
+
+export const RemainingAboutLabels = ({ user }: Props) => (
+  <>
+    <LabelAndText
+      label={AGE_GENDER}
+      text={`${user.age} / ${user.gender} ${
+        user.pronouns ? `(${user.pronouns})` : ""
+      }`}
+    />
+    <LabelAndText
+      label={LANGUAGES_FLUENT}
+      text={
+        user.languagesList.toString().replace(",", ", ") ||
```

Comment:
```suggestion
        user.languagesList?.join(", ") ||
```
Not sure if it needs ? or not

Reply:
I dont think so because languagesList is an array. So if we have user we'll have user.languagesList === [ ], which isn't falsy.
I could do user?.languagesList but I don't think thats needed since we have the ||

Reply:
Sorry I did indeed mean `user?.languagesList`.